entrypoint.sh

```diff
@@ -1,40 +1,64 @@
 #!/bin/sh
 #
-# Sync/Diff rules to a Cortex cluster using the cortex tool
+# Interact with the Cortex Ruler API using the cortextool
 
 err() {
   echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: " + "$@" >&2
 }
 
+# For commands that interact with the server, we need to verify that the
+# CORTEX_TENANT_ID and CORTEX_ADDRESS are set.
+verifyTenantAndAddress() {
```

Comment:
Now that all the commands don't interact with the Cortex cluster, these are optional.